db/migrate/20210709042135_create_external_upload_stubs.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class CreateExternalUploadStubs < ActiveRecord::Migration[6.1]
+  def change
+    create_table :external_upload_stubs do |t|
+      t.string :key, null: false
+      t.string :original_filename, null: false
+      t.integer :status, default: 1, null: false, index: true
+      t.string :unique_identifier, null: false, index: true
```

Comment:
Should this be a `uuid` column type instead which seems to be a better data type fit as compared to the string data type. I believe the index created will also be better for performance: https://stackoverflow.com/a/56453329

Reply:
That's interesting the UUIDs are faster! Thanks for that, will change